spec/elements/break_spec.rb

```diff
@@ -31,20 +17,11 @@
     end
   end
 
-  for_attribute(:type) do
-    with_value(:column) do
-      it_should_assign_successfully
-      it_should_output "<w:br w:type=\"column\"/>"
-    end
-
-    with_value(:page) do
-      it_should_assign_successfully
-      it_should_output "<w:br w:type=\"page\"/>"
-    end
-
-    with_value(:textWrapping) do
+  for_attribute(:type, with_namespace: :w) do
+    allowed = %i(column page textWrapping)
+    with_values(allowed) do
       it_should_assign_successfully
-      it_should_output "<w:br w:type=\"textWrapping\"/>"
+      it_should_output_regular_xml
```

Comment:
It seems to me that tests like:

```ruby
it_should_output "<w:br w:type=\"textWrapping\"/>"
```

do a better job of _documentation_ than tests like:

```ruby
it_should_output_regular_xml
```

In the test suite, there can be _too much_ abstraction (!)

What motivated the change?

Do we really need to loop through and test it with every allowed value? If it works for one allowed value, I'd expect it to work for another... But it seems important to test:
  1. Given a disallowed value, it raises and ArgumentError
  2. Given an allowed value, it produces the expected XML

Or: what am I missing? :grin: